### DISCUSSION TITLE
QUESTION: is it possible to set up the rendering of Vue components?

### DISCUSSION THREAD

**Discussion by denisoed** [GrapesJS/grapesjs#3723] [Q&A]
GrapesJs is the best page builder! :heart: 

Sorry, i am new to GrapesJs.

Can I add support for vue components?

**Comment by artf** [comment]
You can, but you have to create a custom renderer for Vue components.
As a reference, I'd suggest looking at [React components demo](https://codesandbox.io/s/grapesjs-react-components-n6sff)

**Reply by tasiotas** [reply]
Hi @artf , would it be possible to provide working example of using Vue components in Block Manager?
Thank you
